src/tm-threads.c

```diff
@@ -1721,7 +1721,7 @@ TmEcode TmThreadSpawn(ThreadVars *tv)
 {
     pthread_attr_t attr;
     if (tv->tm_func == NULL) {
-        printf("ERROR: no thread function set\n");
+        FatalError(SC_ERR_TM_THREADS_ERROR, "ERROR: no thread function set\n");
```

Comment:
`ERROR:` is redundant here. Not introduced by you, but since you're updating this line please remove. Also get rid of the `\n`